src/go/Makefile

```diff
@@ -101,6 +101,12 @@ define TEST_ENV
 	TEST_MONGODB_FLAVOR=$(TEST_MONGODB_FLAVOR)
 endef
 
+init:                       ## Install linters.
+	go build -modfile=tools/go.mod -o bin/gofumports mvdan.cc/gofumpt/gofumports
+	go build -modfile=tools/go.mod -o bin/gofumpt mvdan.cc/gofumpt
+	go build -modfile=tools/go.mod -o bin/golangci-lint github.com/golangci/golangci-lint/cmd/golangci-lint
+	go build -modfile=tools/go.mod -o bin/reviewdog github.com/reviewdog/reviewdog/cmd/reviewdog
```

Comment:
That can be implemented via `go:generate`
https://github.com/percona/pmm-managed/blob/222b46e62e9978a41b1f5eca39d20878ba6444e1/tools/tools.go#L37-L47

In that case you will have all tools settings in one package. But decision in up to you.